release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -276,6 +277,11 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 [id="ocp-4-10-auth"]
 === Authentication and authorization
 
+[id="ocp-4-10-oc-commands-obtain-credentials-from-podman-config"]
+==== `oc` commands now obtain credentials from podman configuration locations
+
+Previously, `oc` commands that use the registry configuration, for example `oc login` or `oc image`, obtained credentials from Docker configuration locations. With {product-title} {product-version}, if a registry entry cannot be found in the default Docker configuration location, `oc` commands obtain the credentials from podman configuration locations. You can set your preference to either Docker or podman by using the `REGISTRY_AUTH_PREFERENCE` environment variable or the `--registry-config` flag.
```

Comment:
We generally capitalize Podman as a noun and use lowercase to describe the `podman` command.

```suggestion
==== `oc` commands now obtain credentials from Podman configuration locations

Previously, `oc` commands that use the registry configuration, for example `oc login` or `oc image`, obtained credentials from Docker configuration locations. With {product-title} {product-version}, if a registry entry cannot be found in the default Docker configuration location, `oc` commands obtain the credentials from podman configuration locations. You can set your preference to either Docker or Podman by using the `REGISTRY_AUTH_PREFERENCE` environment variable or the `--registry-config` flag.
```